### ISSUE TITLE
` += ` shall not be used before ` ?= ` assignment

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#167]
```
A += "b"
A ?= "a"
```
is bad, while
```
A ?= "a"
A += "b"
```
does what's expected. As a followup to #160, this should be warned about, but only if the ` += ` operator is used *before* the weak default.

The same applies to the following operations ` .= `, ` =+ `, ` =. `
